js/index.js

```diff
@@ -69,9 +69,11 @@ const getMemberImgs = () => {
 
 if (doesGitHubCookieExist()) {
   window.addEventListener('DOMContentLoaded', fetchData);
-} else {
-  window.addEventListener('DOMContentLoaded', updateGitHubLink);
 }
+// the code redirects the user to fake page , commeting it to prevent user for redirection
+// else {
+//   window.addEventListener('DOMContentLoaded', updateGitHubLink);
+// }
```

Comment:
Please don't keep this unused code

No need to comment it out just remove it

Reply:
Hey @RitikJaiswal75  I have removed the commented code please review and approve if its fine .